.ci/orka/install.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -xe
+export PATH=/opt/homebrew/bin:/usr/local/bin:$PATH
+
+MACHINE_TYPE=$(uname -m)
+
+if [ ${MACHINE_TYPE} == "x86_64" ]; then
+    echo "Running on x86_64"
+
+    PYTHON3_URL=https://www.python.org/ftp/python/3.10.8/python-3.10.8-macos11.pkg
+    PYTHON3_FILE=python-3.10.8-macos11.pkg
```

Comment:
I'd pull the variables that are identical for all machine types out of this if statement.